orttraining/orttraining/python/training/ortmodule/_io.py

```diff
@@ -449,8 +449,8 @@ def _add_input(name, input, onnx_graph, onnx_graph_input_names):
                       dynamic_axes=dynamic_axes,
                       schema=schema,
                       num_positionals=len(inputs),
-                      num_positionals_non_none=len([i for i in inputs if i is not None]),
-                      keyword_names=kwargs.keys())
+                      num_positionals_non_none=len(list(i for i in inputs if i is not None)),
+                      keyword_names=list(v for v in kwargs.keys()))
```

Comment:
What is the intent of changes to these two lines? It seems to be doing exactly the same thing before and after the change.

Reply:
I will change them back.